### PR TITLE
feat: added auto dependencies update via DependaBot

### PR DESCRIPTION
**Pull request by mmornati**

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
```
Comment (rdelgatte) [inline .dependabot/config.yml]: Shouldn't we do it in a daily basis?

Comment (lusoalex) [reply .dependabot/config.yml]: It will be done on each push, I do prefer this way to be informed as soon as i push...

Comment (mmornati) [reply .dependabot/config.yml]: Yes we can if you prefer. Don't know what was the difference between `live` and `daily` (even if I can understand what daily means :) @lusoalex , do you know it?

Comment (mmornati) [reply .dependabot/config.yml]: So if it is in each push this means that you won't have PR if you are not working on the project and you never know if there are security updates. So Remi is right, maybe daily is better.

Can we mix live and add a daily only for security updates?

**Comment by mmornati** [comment]
What do we do for this PR in the end? `live` or `daily`?

File: .dependabot/config.yml (hunk 1 above)
Comment (rdelgatte) [reply .dependabot/config.yml]: I would let it as daily because we need to review the PR anyway ;-)

Comment (rdelgatte) [reply .dependabot/config.yml]: As discussed with @mmornati we will leave it to `live` for now. wait and see.

**Pull request merged**